tests/js/spec/components/deviceName.spec.jsx

```diff
@@ -0,0 +1,51 @@
+import * as Sentry from '@sentry/react';
+import {render, screen, waitFor} from '@testing-library/react';
+import * as devicelist from 'ios-device-list';
+
+import * as Device from 'sentry/components/deviceName';
```

Comment:
```suggestion
import {DeviceName} from 'sentry/components/deviceName';
```

Reply:
We need this for the `renders device name if module fails to load` case. I basically mock a module loading failure